### PR TITLE
[MB-3646] Domestic Unpack Pricer

### PR DESCRIPTION
**Pull request by pearl-truss**
## Description

This PR creates a pricer for the unpacking (DUPK) service item.

## Reviewer Notes

This PR uses the MTO shipment weight, following this slack thread, https://ustcdp3.slack.com/archives/CP6F568DC/p1596579174338300. It's noted that in the future this will likely need to change.

## Setup

```sh
make db_dev_e2e_populate
make server_run
```

You'll need to:

-  Create or find a MTO with a shipment that includes the DUPK service item
- Update the MTO to set a primeActualWeight,  scheduledPickupDate and actualPickupDate
- Once you've created a payment request via the prime API verify that the pricing received is equal to the value found manually with this formula

`priceInCents = (Weight Billed Actual / 100) * rateInCents * escalationCompounded`

`rateInCents` and `escalationCompounded` can be found from running this query. Not the dop.schedule value should match the `ServicesScheduleOrigin` value that was returned from the api with create payment request in the previous steps. 

```sql
select price_cents, escalation_compounded from re_domestic_other_prices dop
   inner join re_services on dop.service_id = re_services.id
   inner join re_contracts on re_contracts.id = dop.contract_id
   inner join re_contract_years on re_contracts.id = re_contract_years.contract_id
   where re_services.code = 'DUPK'
    and re_contracts.code = 'TRUSS_TEST'
    and dop.is_peak_period = true
    and dop.schedule = 1
    and '2020-08-31' between re_contract_years.start_date and re_contract_years.end_date;
```
More general acceptance documentation can be found here: https://github.com/transcom/mymove/wiki/Acceptance-Testing-Payment-Requests#domestic-unpacking-price-dupk

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

### CODE REVIEW

File: pkg/services/ghcrateengine/domestic_pack_pricer_test.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ const (
 )
 
 func (suite *GHCRateEngineServiceSuite) TestPriceDomesticPackWithServiceItemParamsBadData() {
-	suite.setUpDomesticPackData(models.ReServiceCodeDPK)
+	suite.setUpDomesticPackUnpackData(models.ReServiceCodeDPK)
```
Comment (jacquelineIO) [inline pkg/services/ghcrateengine/domestic_pack_pricer_test.go:20]: Did you mean to change these?

File: pkg/services/ghcrateengine/domestic_pack_pricer_test.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ func (suite *GHCRateEngineServiceSuite) TestPriceDomesticPackWithServiceItemPara
 }
 
 func (suite *GHCRateEngineServiceSuite) TestPriceDomesticPackWithServiceItemParams() {
-	suite.setUpDomesticPackData(models.ReServiceCodeDPK)
+	suite.setUpDomesticPackUnpackData(models.ReServiceCodeDPK)
```
Comment (jacquelineIO) [inline pkg/services/ghcrateengine/domestic_pack_pricer_test.go:57]: Did you mean to change these?

File: pkg/services/ghcrateengine/ghc_rate_engine_service_test.go [hunk 3]
```diff
@@ -88,7 +88,7 @@ func (suite *GHCRateEngineServiceSuite) setupTaskOrderFeeData(code models.ReServ
 	suite.MustSave(&taskOrderFee)
 }
 
-func (suite *GHCRateEngineServiceSuite) setUpDomesticPackData(code models.ReServiceCode) {
+func (suite *GHCRateEngineServiceSuite) setUpDomesticPackUnpackData(code models.ReServiceCode) {
```
Comment (jacquelineIO) [inline pkg/services/ghcrateengine/ghc_rate_engine_service_test.go:91]: Same question here, I guess maybe this just needs to be a generic name or just leave it as `pack`? Or `PackAndUnpack`? (also not a big deal... but a little off putting)

Comment (pearl-truss) [reply pkg/services/ghcrateengine/ghc_rate_engine_service_test.go:91]: Oh yeah `PackAndUnpack` sounds much better, I will update

**Pull request merged**